GeoIP2/DatabaseReader.cs

```diff
@@ -39,9 +39,9 @@ public DatabaseReader(string file, FileAccessMode mode = FileAccessMode.MemoryMa
         /// <param name="file">The MaxMind DB file.</param>
         /// <param name="locales">List of locale codes to use in name property from most preferred to least preferred.</param>
         /// <param name="mode">The mode by which to access the DB file.</param>
-        public DatabaseReader(string file, List<string> locales, FileAccessMode mode = FileAccessMode.MemoryMapped)
+        public DatabaseReader(string file, IEnumerable<string> locales, FileAccessMode mode = FileAccessMode.MemoryMapped)
```

Comment:
@eilara, public signatures should be using IEnumerable (although I'll re-review). I didn't bother with protected ones, but maybe I should.

Re: test, the deserialization test now tests this (as it doesn't explicitly set the locale anymore).